harness/determined/exec/gc_checkpoints.py

```diff
@@ -126,10 +126,10 @@ def main(argv: List[str]) -> None:
         help="Glob list to match against checkpoint list (JSON-formatted file)",
     )
     parser.add_argument(
-        "--delete-tensorboards",
+        "--deleted-experiment",
```

Comment:
Maybe the flag could be called ```should-patch-checkpoints```?

Reply:
+1 to `patch-checkpoints`, only because if we do expand the tensorboard deletion project & create an individual endpoint for it, i'd say we should name _that_ endpoint/all its related code `delete-tensorboards`